packages/components/molecules/f-rating/src/components/Rating.vue

```diff
@@ -2,30 +2,97 @@
     <div
         :class="$style['c-rating']"
         data-test-id="rating-component">
-        {{ copy.text }}
+        <ul :class="$style['c-rating-star-wrapper']">
+            <li
+                v-for="star in maxStarRating"
+                :key="star"
+                :class="$style['c-rating-star']">
+                <star-filled-icon
+                    v-if="hasRating(star)"
+                    :class="$style['c-rating-star-filled']" />
+                <star-icon
+                    v-else
+                    :class="$style['c-rating-star-empty']" />
+            </li>
+        </ul>
+        <span
```

Comment:
could this be an `aria` attribute (such as label) on the `ul` element itself?

Reply:
That's a good suggestion. I'll have a think and add that.

Reply:
I've had a play with this and it turns out (on Mac anyway) voiceover works much better with how it is right now. Using an aria label doesn't really work as good. Thoughts?

Reply:
Sorry I only just saw this - yep, if the UX is preferable using the current approach then go for it! Out of interest what was the difference?

Reply:
I'm going to take a look at this separately as i will look to improve this in the next ticket. :)